src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/DeclarationOrderCheck.java

```diff
@@ -84,41 +84,112 @@
  * </li>
  * </ul>
  * <p>
- * To configure the check:
+ * To configure the check with default config:
```

Comment:
` with default config`
no need for it, many checks have it in short version, so no need to clarify it

Reply:
Done.